test/prop_dispcount.erl

```diff
@@ -75,7 +76,26 @@ prop_nocrash() ->
                                 R =:= ok))
         end).
 
-prop_parallel_nocrash() ->
+prop_nocrash_ets_hash() ->
+    prop_nocrash(ets, hash).
+
+prop_nocrash_ets_round_robin() ->
+    prop_nocrash(ets, round_robin).
+
+prop_nocrash_named_hash() ->
+    prop_nocrash(named, hash).
+
+prop_nocrash_named_round_robin() ->
+    prop_nocrash(named, round_robin).
+
+prop_nocrash_atomics_hash() ->
+    prop_nocrash(atomics, hash).
+
+prop_nocrash_atomics_round_robin() ->
+    prop_nocrash(atomics, round_robin).
+
+%% this seems broken?
+noprop_parallel_nocrash() ->
```

Comment:
possibly? I don't think I truly understood stateful tests back then. Seems to work as a suite though.

Reply:
I couldn't figure out how to run it in a way that didn't make it fail, pointers welcome.